castle/cms/files/aws.py

```diff
@@ -20,6 +19,7 @@
 from zope.component import getUtility
 
 
+install_aliases()  # noqa
```

Comment:
this should go back to its original position so that future.standard_library aliases can be correctly applied before other imports are declared